encryption/caesar.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include "caesar.h"
+
+int euclideanModulo(int a, int b) {
+    return ((a) < 0 ? ((((a) % (b)) + (b)) % (b)) : ((a) % (b)));
+}
+
+namespace encryption {
+
+    std::string Caesar::encrypt(std::string userInput) {
```

Comment:
Let's think about avoid to copy string here, maybe let's try to use const reference or move value, if only this method will use original string ?